src/middlewared/middlewared/plugins/device_/gpu_linux.py

```diff
@@ -0,0 +1,33 @@
+import re
+
+from middlewared.service import Service
+from middlewared.utils import run
+
+
+RE_VENDOR = re.compile(r'description:\s*VGA compatible controller[\s\S]*vendor:\s*(.*)')
+
+
+class DeviceService(Service):
+
+    GPU = None
+
+    async def available_gpu(self):
```

Comment:
This is not using the same API...
I meant adding a `GPU` entry to device.get_info